ci/no-std-check/Cargo.toml

```diff
@@ -15,10 +15,10 @@ tendermint = { version = "0.34", default-features = false }
 tendermint-proto = { version = "0.34", default-features = false }
 tendermint-light-client-verifier = { version = "0.34", default-features = false, features = ["rust-crypto"] }
 
-sp-core = { version = "24.0.0", default-features = false, optional = true }
-sp-io = { version = "26.0.0", default-features = false, optional = true }
-sp-runtime = { version = "27.0.0", default-features = false, optional = true }
-sp-std = { version = "11.0.0", default-features = false, optional = true }
+sp-core = { version = "25.0.0", default-features = false, optional = true }
+sp-io = { version = "27.0.0", default-features = false, optional = true }
+sp-runtime = { version = "28.0.0", default-features = false, optional = true }
+sp-std = { version = "12.0.0", default-features = false, optional = true }
```

Comment:
Why are only these dependencies upgraded, but not others?

Reply:
[eff17e4](https://github.com/cosmos/ibc-rs/pull/949/commits/eff17e48da37dfad29e2da9f8993368510badc88)